apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/model/entity/OrganizationEntityTest.java

```diff
@@ -0,0 +1,227 @@
+    String retrievedWebsiteUrl = organizationEntity.getWebsiteUrl();
+    Integer retrievedChallengeCount = organizationEntity.getChallengeCount();
+    List<OrganizationCategoryEntity> retrievedCategories = organizationEntity.getCategories();
+    List<ChallengeContributionEntity> retrievedChallengeContributions =
+        organizationEntity.getChallengeContributions();
+    String retrievedDescription = organizationEntity.getDescription();
+    OffsetDateTime retrievedCreatedAt = organizationEntity.getCreatedAt();
+    OffsetDateTime retrievedUpdatedAt = organizationEntity.getUpdatedAt();
+    String retrievedAcronym = organizationEntity.getAcronym();
+
+    Assertions.assertEquals(id, retrievedId);
+    Assertions.assertEquals(name, retrievedName);
+    Assertions.assertEquals(email, retrievedEmail);
+    Assertions.assertEquals(login, retrievedLogin);
+    Assertions.assertEquals(avatarKey, retrievedAvatarKey);
+    Assertions.assertEquals(websiteUrl, retrievedWebsiteUrl);
+    Assertions.assertEquals(challengeCount, retrievedChallengeCount);
+    Assertions.assertEquals(Collections.emptyList(), retrievedCategories);
+    Assertions.assertEquals(Collections.emptyList(), retrievedChallengeContributions);
+    Assertions.assertEquals(description, retrievedDescription);
+    Assertions.assertEquals(createdAt, retrievedCreatedAt);
+    Assertions.assertEquals(updatedAt, retrievedUpdatedAt);
+    Assertions.assertEquals(acronym, retrievedAcronym);
+  }
+
+  @Test
+  public void HashCode_ShouldBuildHashCodeOfArgumentsForOrganizationEntity_WhenArgumentsPassed() {
+
+    OffsetDateTime createdAt = OffsetDateTime.now();
+    OffsetDateTime updatedAt = OffsetDateTime.now();
+
+    OrganizationEntity organizationEntity = new OrganizationEntity();
+
+    organizationEntity.setId(1L);
+    organizationEntity.setName("Test Organization");
+    organizationEntity.setEmail("test@example.com");
+    organizationEntity.setLogin("testorg");
+    organizationEntity.setDescription("Test description");
+    organizationEntity.setAvatarKey("avatarKey");
+    organizationEntity.setWebsiteUrl("https://example.com");
+    organizationEntity.setChallengeCount(5);
+    organizationEntity.setCreatedAt(createdAt);
+    organizationEntity.setUpdatedAt(updatedAt);
+    organizationEntity.setAcronym("TO");
+
+    OrganizationEntity organizationEntity2 = new OrganizationEntity();
+    organizationEntity2.setId(1L);
+    organizationEntity2.setName("Test Organization");
+    organizationEntity2.setEmail("test@example.com");
+    organizationEntity2.setLogin("testorg");
+    organizationEntity2.setDescription("Test description");
+    organizationEntity2.setAvatarKey("avatarKey");
+    organizationEntity2.setWebsiteUrl("https://example.com");
+    organizationEntity2.setChallengeCount(5);
+    organizationEntity2.setCreatedAt(createdAt);
+    organizationEntity2.setUpdatedAt(updatedAt);
+    organizationEntity2.setAcronym("TO");
```

Comment:
This is a good opportunity to start using a "@BeforeAll" (see this [article](https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall)) to define objects ONCE and then use them in more than one test.

Reply:
Added `@BeforeAll` to the code to initialized the properties of each organization entity that is generated.